docs/installing_deis/azure.rst

```diff
@@ -0,0 +1,121 @@
+while following this documentation.
+
+
+Install Python and Azure SDK for Python
+---------------------------------------
+
+The cluster creation tool uses Python and the Python Azure library to create a CoreOS cluster.
+If you haven't already, install these on your development machine:
+
+.. code-block:: console
+
+    $ brew install python
+    $ sudo pip install azure
+
+And check to make sure they are configured correctly:
+
+.. code-block:: console
+
+    $ python -c "import azure; print(azure.__version__)"
+    0.9.0  <-- everything working ok!
+
+Generate Certificates
+---------------------
+
+The azure-coreos-cluster creation tool uses the Azure management REST API to create the CoreOS
+cluster which uses a management certificate to authenticate.
+
+If you don't have a management certificate already configured, the script generate-mgmt-cert.sh can
+create this certificate for you. Otherwise, you can skip to the next section.
+
+If you need to create a certificate, edit cert.conf in contrib/azure with your company's details and then run:
+
+.. code-block:: console
+
+    $ ./generate-mgmt-cert.sh
+
+Upload Management Cert
+----------------------
+
+If you haven't uploaded your management certificate to Azure (azure-cert.cer if you used the script
+in the previous section), do that now using the `management certificates tab`_ of the
+Azure portal's Settings.
+
+Also copy the Azure subscription id from this table and save it for the cluster creation script below.
+
+Create Cluster Cloud Config
+---------------------------
+
+Before we can create a cluster, we need to create a cloud config for it. The script
+``create-azure-user-data`` does this for you. This script takes the stock cluster instance config
+in ``../coreos/user-data.example``, customizes it for Azure, and inserts a unique cluster discovery
+endpoint:
+
+.. code-block:: console
+
+    $ ./create-azure-user-data $(curl -s https://discovery.etcd.io/new)
+
+This will create a azure-user-data cloud config file. We'll use this with the script in the next
+section during cluster creation.
+
+Create CoreOS Cluster
+---------------------
+
+With the management certificate and cloud config in place, we are ready to create our cluster.
+
+* Create a container called ``vhds`` within a storage account in the same region as your cluster using the Azure portal. Note the URL of the container for the cluster creation script below.
+* Choose a cloud service name for your Deis cluster for the script below. The script will automatically create this cloud service for you.
+* Choose an Azure `region`_ to use. Supply it in quotes with the ``--location`` parameter. The default is "West US".
+
+With that, let's run the azure-coreos-cluster script which will create the CoreOS cluster. Fill in the bracketed values with the values for your deployment you created above.
+
+.. code-block:: console
+
+    $ ./azure-coreos-cluster [cloud service name]
```

Comment:
Should we use `--ssh-cert` and `--ssh-thumb` to point to `~/.ssh/deis.key`, similar to our other scripts?